### PR TITLE
Adds concepts to publication solr docs.

### PR DESCRIPTION
**Pull request by justinlittman**
refs https://github.com/sul-dlss-labs/rialto-webapp/issues/96

### CODE REVIEW

File: repository/sparql_reader.go [hunk 1]
```diff
@@ -148,7 +148,19 @@ func (r *SparqlReader) GetAuthorInfo(publication string) (*sparql.Results, error
 				 ?authorship <http://vivoweb.org/ontology/core#relates> ?author .
 				 ?author <http://www.w3.org/2004/02/skos/core#prefLabel> ?author_label .
 			}
-			ORDER BY ?org OFFSET 0 LIMIT 100`, publication)
+			ORDER BY ?author OFFSET 0 LIMIT 100`, publication)
```
Comment (mjgiarlo) [inline repository/sparql_reader.go]: Can you say more about this change to the order?

File: repository/sparql_reader.go [hunk 2]
```diff
@@ -148,7 +148,19 @@ func (r *SparqlReader) GetAuthorInfo(publication string) (*sparql.Results, error
...
+	return r.repo.Query(query)
+}
+
+// GetConceptInfo retrieves a list of concepts for a subject
+func (r *SparqlReader) GetConceptInfo(subject string) (*sparql.Results, error) {
+	query := fmt.Sprintf(`SELECT ?id ?label
+		 WHERE {
+         <%s> <http://purl.org/dc/terms/subject> ?id .
+				 ?id a <http://www.w3.org/2004/02/skos/core#Concept> .
```
Comment (mjgiarlo) [inline repository/sparql_reader.go]: The indentation got funky here.

File: repository/sparql_reader.go (hunk 1 above)
Comment (justinlittman) [reply repository/sparql_reader.go]: `?org` was a mistake. There's no org variable in the query.

File: repository/sparql_reader.go (hunk 2 above)
Comment (justinlittman) [reply repository/sparql_reader.go]: De-funked, I think.

**Pull request merged**